tests/components/grid_manipulation.test.ts

```diff
@@ -150,6 +151,20 @@ describe("Context Menu add/remove row/col", () => {
     expect(fixture.querySelector(".o-menu div[data-name='hide_rows']")).toBeNull();
   });
 
+  test("cannot delete all non-frozen rows with contextmenu when frozen panes exist", async () => {
```

Comment:
The test should be in `menu_items_registry.test.ts`

Reply:
Since the `cannot delete nor hide all rows with contextmenu` test case is written here, that's why I have included my test case here. [TESTCASE](https://github.com/odoo/o-spreadsheet/blob/34011c3de11b883dbd9ca0494d3ff8e09e5b58c8/tests/components/grid_manipulation.test.ts#L123-L151)

Should we consider adding this to the menu_items_registry.test.ts file?